include/Capabilities/Capabilities.php

```diff
@@ -13,6 +13,9 @@
  * @since 3.8
  */
 class Capabilities {
+	public const LANGUAGES_CAPABILITY    = 'manage_languages';
+	public const TRANSLATIONS_CAPABILITY = 'manage_translations';
```

Comment:
Given the "Capabilities" is already included in the class name, I propose to simplify:
```suggestion
	public const LANGUAGES    = 'manage_languages';
	public const TRANSLATIONS = 'manage_translations';
```

Reply:
Done in [f44c792](https://github.com/polylang/polylang/pull/1744/commits/f44c79219ce66ea8d46b3c165db67b62d28948d6).